### DISCUSSION TITLE
Release date SDK 1.9.0

### DISCUSSION THREAD

**Discussion by OliverStingl** [OP-TED/eForms-SDK#673] [Q&A] [answered]
Hello!

Do we already have a release date for SDK 1.9.0?
I couldn't find any information on the website or in the forums.

**Comment by rouschr** [comment, marked as answer]
Hello, 
as a developer I cannot promise a precise date (as this is too political for my role), but I can tell you that we are still incorporating some "last feedback" into 1.9, in parallel we already work on SDK 1.10 and we also patch older SDK versions.
From my perspective the main feature of 1.9 will be the addition of fields representing XML attributes and their link to the XML element fields (which have the attributes), this will help with form creation and XML generation.
In the mean time you can have a look at the release candidate: 
https://github.com/OP-TED/eForms-SDK/tree/1.9.0-rc.1

**Reply by OliverStingl** [reply]
Thank you very much!
